pubspec.yaml

```diff
@@ -1,23 +1,24 @@
 name: github
-version: 7.0.4
+version: 7.1.0-nullsafe.0
```

Comment:
We usually consider these breaking changes. It's the safest thing to do!

Reply:
Can do